template/koa-rest-api/src/tracing.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Opentelemetry tracing initialisation. This is a standalone ts/js module that is not
+ * referenced by application source code directly. It is required at runtime using the
+ * node command's `--require` argument, see Dockerfile for details.
+ */
+
+import { propagation } from '@opentelemetry/api';
+import { CompositePropagator } from '@opentelemetry/core';
+import { CollectorTraceExporter } from '@opentelemetry/exporter-collector-grpc';
+import { AwsInstrumentation } from '@opentelemetry/instrumentation-aws-sdk';
+import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
+import { B3InjectEncoding, B3Propagator } from '@opentelemetry/propagator-b3';
+import { NodeSDK } from '@opentelemetry/sdk-node';
+
+const main = async () => {
+  // Use B3 propagation to ensure proper propagation between systems use OpenTelemetry and
+  // native Datadog APM, such as Istio/Envoy
+  propagation.setGlobalPropagator(
+    new CompositePropagator({
+      propagators: [
+        new B3Propagator(),
+        new B3Propagator({ injectEncoding: B3InjectEncoding.MULTI_HEADER }),
+      ],
+    }),
+  );
+
+  const sdk = new NodeSDK({
+    traceExporter: new CollectorTraceExporter(),
+    autoDetectResources: false,
+    instrumentations: [new HttpInstrumentation(), new AwsInstrumentation()],
+  });
+
+  return sdk.start();
+};
+
+const log = (level: string, msg: string, extra = {}) => {
```

Comment:
I'm hesitant to bring in the standard logger as it might bring more dependencies before internals instrimented. This lib really meant to be self-contained.

Thoughts?